src/explore-education-statistics-admin/src/pages/release/content/contexts/ReleaseContentContext.tsx

```diff
@@ -86,15 +83,19 @@ export const releaseReducer: Reducer<
             contentBlock => contentBlock.id === blockId,
           );
 
-          matchingContentSection.content[blockIndex] = block;
+          matchingContentSection.content[blockIndex] = block ?? {
+            ...matchingContentSection.content[blockIndex],
+          };
```

Comment:
Why are we doing this? Shouldn't `block` be defined? Feels like we  shouldn't even get this far i.e. the action should never be triggered